README.md

```diff
@@ -1,96 +1,179 @@
-
-
-Other documents
-===============
+Remote monitoring helps you get better visibility into your devices, assets, and
+sensors wherever they happen to be located. You can collect and analyze real-time
+device data using a remote monitoring solution that triggers automatic alerts and
+actions — from remote diagnostics to maintenance requests.
 
-* [Contributing](CONTRIBUTING.md)
-* [Development setup](DEVELOPMENT.md)
+[Azure IoT Hub](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-what-is-iot-hub)
+is a key building block of the remote monitoring sultion. IoT Hub is a fully
+managed service that enables reliable and secure bidirectional communications between
+millions of IoT devices and a solution back end.
 
+Check out the [Interactive Demo](http://dev-azureiotclicks01.azurewebsites.net/demos/remotemonitoring)
+for a detailed overview of features and use cases.
 
-[build-badge]: https://img.shields.io/travis/Azure/azure-iot-pcs-remote-monitoring-java.svg
-[build-url]: https://travis-ci.org/Azure/azure-iot-pcs-remote-monitoring-java
-[issues-badge]: https://img.shields.io/github/issues/azure/azure-iot-pcs-remote-monitoring-java.svg
-[issues-url]: https://github.com/azure/azure-iot-pcs-remote-monitoring-java/issues
+To get started you can folow along with the [Getting Started](#getting-started)
+for a command line deployment. You can also deploy using the web interface
+at https://azureiotsuite.com.
+
+>  This solution is currently in preview. The privew offers many features
+but there are known bugs. You can refer to our [Issues List](issues) in
+this repo (and in any submodules) to see known requests and issues. Feel
+free to submit new issues (and submit PRs for fixes if you'd like).
+
+Getting Started
+===============
+
+## Prerequisites
+* Azure Subscription ([permissions guidelines](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-permissions))
+
+## Setup
+1. Check out the repository and submodules
+    ```
+    git clone --recursive https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet.git
+    ```
+1. Set up command line interface for deployments
+    ```
+    cd pcs-cli
+    npm install
+    npm start
+    npm link
+    ```
+1. From `pcs-cli`, sign into your account for pcs-cli with `pcs login`
+
+## Standard Deployment
+```
+pcs -s standard --runtime java
+```
+
+## Basic Deployment
+```
+pcs --runtime java
+```
+
+## Local Deployment
+To set up a Remote Monitoring Solution locally see
+[Running all PCS microservices locally](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#running-all-pcs-microservices-locally)
+
+Common Scenarios
+================
+
+## All done? Connect a device!
+By default the solution uses simulated devices. You can start adding your
+own devices with the instructions here [Connect a physical device](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#connect-a-physical-device)
+
```

Comment:
You can also stop simulation either from the UI [screenshot of UI], or stop the simulation microservice directly by doing X [link to developer guide for docker stop <container>.